pkg/test/framework/components/istio/cleanup.go

```diff
@@ -109,6 +111,19 @@ func (i *istioImpl) cleanupCluster(c cluster.Cluster, errG *multierror.Group) {
 		if e := i.installer.Close(c); e != nil {
 			err = multierror.Append(err, e)
 		}
+
+		err = multierror.Append(err, retry.UntilSuccess(func() error {
```

Comment:
just selecting a shorter `terminationGracePeriodSeconds` is probably enough to fix the post submit job - but this just seems like a good idea to prevent duplicate failures in the future

Reply:
the wait time for this check I currently have is based on `terminationGracePeriodSeconds` for ztunnel, so if there are opinions on the duration I am happy to adjust

Reply:
Can you explain why this is important? We intentionally do not wait for things like this in other parts of the test since its slow.

If we do want to do this, why don't we just force kill ztunnel? There isn't much point waiting for it to take forever to shutdown

Reply:
Discussed offline. https://github.com/istio/istio/blob/c2634f83522651692f58d3cfe858491be7257ed7/pkg/test/framework/components/namespace/kube.go#L237 causes istio-system namespace to be deleted unexpectedly; we should fix that and this problem goes away I think